OpenUpSA/wazimap-ng-ui
# adding a link to direct to the product page

## Description
Added a link to `Wazimap` product page

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-152

## How is it tested automatically?

## How should a reviewer test it locally
* confirm that on the right-hand side, there is a link and when you click on the link, you get directed to the `Wazimap` product page 

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/145101569-ee89f18d-7ecc-45c8-be9d-7730c93c8682.png)

## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
